src/components/MapCanvas.js

```diff
@@ -1,11 +1,28 @@
 import React, { Component } from 'react';
-import { Header } from 'semantic-ui-react';
+import ReactMapGL, { Marker } from 'react-map-gl';
+import styled from 'styled-components/macro';
+import LocationIcon from '@material-ui/icons/LocationOn';
 
 import summary from '../apis/summary';
 
+const MAPBOX_TOKEN =
+  'pk.eyJ1Ijoibmlja2hvYW5nIiwiYSI6ImNqeDB3NzNzbjAzamg0Ym83aXZmcWEzbDcifQ.hvWWVoMPsWS2KVa8nFfOkA'; //Will be moved to env file, leave here for testing
```

Comment:
This is something you don't want in the commit history. We should move this to an env file as soon as possible, and then squish the commit so that no one can find this key.